src/giftCards/GiftCardUpdate/GiftCardUpdatePageHeader/hooks/useGiftCardActivationDeactivation.ts

```diff
@@ -0,0 +1,92 @@
+import useNotifier from "@saleor/hooks/useNotifier";
+import commonErrorMessages from "@saleor/utils/errors/common";
+import { useIntl } from "react-intl";
+
+import { giftCardEnableDisableSectionMessages as messages } from "../messages";
+import {
+  useGiftCardActivateMutation,
+  useGiftCardDeactivateMutation
+} from "../mutations";
+import { GiftCardActivate } from "../types/GiftCardActivate";
+import { GiftCardDeactivate } from "../types/GiftCardDeactivate";
+
+interface UseGiftCardActivationDeactivationProps {
+  onActivateActionComplete?: () => void | undefined;
+  onDeactivateActionComplete?: () => void | undefined;
```

Comment:
Same as?
```suggestion
  onActivateActionComplete?: () => void;
  onDeactivateActionComplete?: () => void;
```

Reply:
I'd also assume these are not optional so remove the `?`